recipes/openssl/1.x.x/conanfile.py

```diff
@@ -74,6 +74,7 @@ class OpenSSLConan(ConanFile):
                "shared": [True, False],
                "fPIC": [True, False],
                "no_asm": [True, False],
+               "enable_weak_ssl_ciphers": [True, False],
```

Comment:
This is just adding that option, but you will need to add the default value:

```python
    default_options = {key: False for key in options.keys()}
    default_options["fPIC"] = True
    default_options["openssldir"] = None
```

and then you will need to pass the proper information to the build system

Reply:
All options are defaulted to `False`:
```python
default_options = {key: False for key in options.keys()}
```
In this case the default matches that of openssl, similar to the rest of `no-xxxx` options used in the recipe.

In regards to passing the information to the build system, as with the `no-xxxx` options, this is picked up in the `_configure_args` property: https://github.com/conan-io/conan-center-index/blob/ab529c5d14a860fc7c569478e4e321e753d539ad/recipes/openssl/1.x.x/conanfile.py#L519-L523
which is then passed on to openssl's Configure.

Reply:
Oh! I thought your intention in the PR was to enable it by default. My fault.

Reply:
Oh no, wouldn't wanna force on everyone an option disabled by default in openssl. Thanks for the review!